libbpf-tools/tcprtt.c

```diff
@@ -139,6 +144,20 @@ static error_t parse_arg(int key, char *arg, struct argp_state *state)
 		}
 		env.raddr = addr.s_addr;
 		break;
+	case 'c':
+		if (inet_pton(AF_INET6, arg, &addr_v6) < 1) {
+			fprintf(stderr, "invalid local IPv6 address: %s\n", arg);
+			argp_usage(state);
+		}
+		memcpy(env.laddr_v6, &addr_v6, sizeof env.laddr_v6);
```

Comment:
please use `sizeof(env.laddr_v6)`. The same for other instances in this file. Just do a search for 'sizeof' and make changes to all those not 'sizeof(...)' cases.